### PR TITLE
More expression conversions 8

### PR DESCRIPTION
**Pull request by mmatera**
the one before the last one...

<a href="https://gitpod.io/#https://github.com/Mathics3/mathics-core/pull/389"></a>

### CODE REVIEW

File: mathics/builtin/list/rearrange.py [hunk 1]
```diff
@@ -424,10 +425,10 @@ class GatherBy(_GatherOperation):
     """
     <dl>
       <dt>'GatherBy[$list$, $f$]'
-      <dd>gathers leaves of $list$ into sub lists of items whose image under $f identical.
+      <dd>gathers elements of $list$ into sub lists of items whose image under $f identical.
```
Comment (rocky) [inline mathics/builtin/list/rearrange.py:428]: Notice a small typo here from before: `$f` -. $f$

File: mathics/builtin/list/rearrange.py [hunk 2]
```diff
@@ -692,7 +693,7 @@ class Riffle(Builtin):
       <dd>inserts a copy of $x$ between each element of $list$.
 
       <dt>'Riffle[{$a1$, $a2$, ...}, {$b1$, $b2$, ...}]'
-      <dd>interleaves the elements of both lists, returning '{$a1$, $b1$, $a2$, $b2$, ...}'.
+      <dd>interelements the elements of both lists, returning '{$a1$, $b1$, $a2$, $b2$, ...}'.
```
Comment (rocky) [inline mathics/builtin/list/rearrange.py:696]: interelments -> inter-elements

**Comment by rocky** [comment]
LGTM

File: mathics/builtin/list/rearrange.py (hunk 2 above)
Comment (mmatera) [reply mathics/builtin/list/rearrange.py:696]: Interleaves?

Comment (rocky) [reply mathics/builtin/list/rearrange.py]: Oops - you are absolutely correct! This is an English word so it should stay as is.

Comment (mmatera) [reply mathics/builtin/list/rearrange.py]: Yep, I changed it accidentally.

**Pull request merged**